src/components/UI/NavbarStyled/Navbar.styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const NavbarStyled = styled.nav`
+	background-color: black;
+	border-top: solid white;
+	width: 100%;
+	bottom: 0;
+	position: fixed;
+	display: flex;
+	align-items: center;
+	justify-content: space-around;
+	padding: 10px;
+	z-index: 1;
+	height: 7vh;
```

Comment:
please don't use this vh version, instead use a fixed px / rem value.

Reply:
Fixed